src/main/java/ch/njol/skript/expressions/ExprWorldBorderWarningDistance.java

```diff
@@ -0,0 +1,72 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.WorldBorder;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Warning Distance of World Border")
+@Description("The warning distance of a world border. The player's screen will be tinted red when they are within this distance of the border")
+@Examples("set warning distance of {_worldborder} to 1")
+@Since("INSERT VERSION")
+public class ExprWorldBorderWarningDistance extends SimplePropertyExpression<WorldBorder, Integer> {
+
+	static {
+		register(ExprWorldBorderWarningDistance.class, Integer.class, "[[world[ ]]border] warning distance", "worldborders");
+	}
```

Comment:
Again not a fan of `world border` not being required especially here where even border isn't